test.cpp

```diff
@@ -241,3 +250,36 @@ int main(int argc, char **argv) {
     std::string points_json = Json(points).dump();
     printf("%s\n", points_json.c_str());
 }
+
+#if(JSON11_TEST_STANDALONE_MAIN)
```

Comment:
Hmm, I always learned to put parens on #if, but not on #ifdef, though I never really knew why.  Might it theoretically matter if the variable expanded to an expression with multiple terms?  Probably not without line breaks involved, so I can't think of a case.

I can remove the parens.